code/CrockerClockFrame.java

```diff
@@ -117,7 +121,14 @@ public void run() {
 						time = 0;
 						ClockDown.cancel(); // stops clock
 					} else {
-						clocky.setText(""+time);//puts word down
+						timeInSecond = time%60; //time in seconds

```

Comment:
Is this redundant?

You already set the timer values at the top. Is this to update the values?

Reply:
any redundancy in the action listen method should be ignored for now as the method does not work as intended I was planning to fix that in my next ticket